data/json/items/book/tailor.json

```
@@ -63,8 +66,8 @@
     "skill": "tailor",
     "max_level": 3,
     "proficiencies": [
-      { "proficiency": "prof_closures", "time_factor": 0.5, "fail_factor": 0.75 },
-      { "proficiency": "prof_elastics", "time_factor": 0.5, "fail_factor": 0.75 }
+      { "proficiency": "prof_closures", "time_factor": 0.2, "fail_factor": 0.25 },
+      { "proficiency": "prof_elastics", "time_factor": 0.1, "fail_factor": 0.25 }
```

```suggestion
      { "proficiency": "prof_elastics", "time_factor": 0.2, "fail_factor": 0.15 }
```
It only takes 18 minutes to read, intelligence required is 4, etc.